server/controllers/excelController.js

```diff
@@ -26,4 +27,580 @@ excelController.read = async (req, res, next) => {
     });
   }
 };
+
+excelController.convertInputs = async (req, res, next) => {
+  console.log('entering excelController.convertInputs')
+
+  // replace with Brian's inputs
+
+  try {
+    res.locals.inputRows = [
+      {
+        name: "Luke Skywalker",
+        mass: 77,
+        hair_color: "blond",
+        skin_color: "fair",
+        eye_color: "blue",
+        birth_year: "19BBY",
```

Comment:
Since I have jest setup on my branch - once my branch merges we can move these to test files and get tests setup!